operators/operator_sdk/osdk-generating-csvs.adoc

```diff
@@ -42,6 +42,10 @@ included in `metadata.name`.
 include::modules/osdk-how-csv-gen-works.adoc[leveloffset=+1]
 include::modules/osdk-csv-composition-configuration.adoc[leveloffset=+1]
 include::modules/osdk-manually-defined-csv-fields.adoc[leveloffset=+1]
+.Additional resources
+
+- xref:../../operators/olm-what-operators-are.adoc#olm-maturity-model_olm-what-operators-are[Operator maturity model]
+
```

Comment:
Isn't this an xref within a module and therefore not allowed? I could be missing something.

Reply:
@bobfuru I believe this is the assembly: operators/operator_sdk/osdk-generating-csvs.adoc

Reply:
`osdk-generating-csvs.adoc` is an assembly, it's just nested under a subdirectory.